src/server.ts

```diff
@@ -1,11 +1,12 @@
 import App from '@/app';
 import AuthRoute from '@routes/auth.route';
+import GroupsRoute from './routes/group.route';
```

Comment:
This works but for more consistency with the rest of the imports it should be @routes instead of ./routes